README.md

```diff
@@ -12,13 +12,8 @@ This package was renamed to `mongodb/laravel-mongodb` because of a transfer of o
 It is compatible with Laravel 10.x. For older versions of Laravel, please refer to the
 [old versions](https://github.com/mongodb/laravel-mongodb/tree/3.9#laravel-version-compatibility).
 
-- [Installation](https://www.mongodb.com/docs/drivers/php/laravel-mongodb/current/install/)
-- [Eloquent Models](https://www.mongodb.com/docs/drivers/php/laravel-mongodb/current/eloquent-models/)
-- [Query Builder](https://www.mongodb.com/docs/drivers/php/laravel-mongodb/current/query-builder/)
-- [Transactions](https://www.mongodb.com/docs/drivers/php/laravel-mongodb/current/transactions/)
-- [User Authentication](https://www.mongodb.com/docs/drivers/php/laravel-mongodb/current/user-authentication/)
-- [Queues](https://www.mongodb.com/docs/drivers/php/laravel-mongodb/current/queues/)
-- [Upgrading](https://www.mongodb.com/docs/drivers/php/laravel-mongodb/current/upgrade/)
+## Documentation
+Read the reference documentation for the Laravel MongoDB package [on MongoDB website](https://www.mongodb.com/docs/drivers/php/laravel-mongodb/).
```

Comment:
I'd suggest copying the syntax we already use in https://github.com/mongodb/mongo-php-library/blob/master/README.md#documentation

```md
## Documentation

 - https://mongodb.com/docs/drivers/php/laravel-mongodb/current/
 - https://www.mongodb.com/docs/drivers/php/
```

Also, I just noticed that https://mongodb.com/docs/ecosystem/drivers/php/ redirects to https://www.mongodb.com/docs/drivers/php/. I'll make that change to the PHPLIB README.